webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -1,32 +1,38 @@
-import React, { useState } from 'react';
-import { Tabs, Tab, TabTitleText, Label } from '@patternfly/react-core';
-import { InfoCircleIcon } from '@patternfly/react-icons';
+import React, { useState } from "react";
+import { Tabs, Tab, TabTitleText, Label } from "@patternfly/react-core";
+import { InfoCircleIcon } from "@patternfly/react-icons";
+import { Route } from "react-router-dom";
+import { translate as __ } from "foremanReact/common/I18n";
 
-import { translate as __ } from 'foremanReact/common/I18n';
+import "./AnsibleHostDetail.scss";
 
-import './AnsibleHostDetail.scss';
-
-const AnsibleHostDetail = props => {
+const AnsibleHostDetail = (props) => {
   // https://projects.theforeman.org/issues/32398
-  const [activeTab] = useState('variables');
+  const [activeTab] = useState("variables");
 
   return (
-    <div className="ansible-host-detail">
+    <>
       <Tabs activeKey={activeTab} isSecondary>
         <Tab
           eventKey="variables"
-          title={<TabTitleText>{__('Variables')}</TabTitleText>}
-        >
-          <Label
-            color="blue"
-            icon={<InfoCircleIcon />}
-            style={{ marginTop: '1.5rem' }}
-          >
-            Ansible Variables coming soon!
-          </Label>
-        </Tab>
+          title={<TabTitleText>{__("Variables")}</TabTitleText>}
+          href="#Ansible/sub-tab"
```

Comment:
Should this be `#Ansible/variables`?

Reply:
it's a poc :) 
if we will decide to use this approach I'll update this